controllers/application_controller.go

```diff
@@ -165,7 +165,7 @@ func (r *ApplicationReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		devfileData, err := devfile.ParseDevfileModel(application.Status.Devfile)
 		if err != nil {
 			log.Error(err, fmt.Sprintf("Unable to parse devfile model, exiting reconcile loop %v", req.NamespacedName))
-			return ctrl.Result{}, err
+			return ctrl.Result{}, nil
```

Comment:
or maybe we dont set the set the status when we return an err and want to reconcile, because that may mean multiple statuses being appended since its an array

Reply:
My concern about not appending a status when we return an err is that if the error keeps occurring, that error will never be surfaced to the user/client.

Reply:
Updated to add the status condition in. It should be there IMO.